internal/transport/transport.go

```diff
@@ -514,6 +501,30 @@ func Truncate(ctx context.Context, cfg *Config) error {
 	return nil
 }
 
+// Truncate will truncate the defined tables in the configuration.
+func Truncate(ctx context.Context, cfg *Config) error {
+	// truncateRequest is a special request that will truncate the table before upserting data.
+	truncateRequest := new(proto.TruncateRequest)
```

Comment:
cases:
1. root-level truncate present
  a. request truncate not present
  b. request truncate present but table not present
2. Request level truncate present
  a. table not present